src/components/SettingsContainer.tsx

```diff
@@ -1,5 +1,4 @@
-import React, { useEffect } from "react";
-import { PropsWithChildren, ReactElement } from "react";
+import React, { Children, PropsWithChildren, ReactElement, useEffect } from "react";
```

Comment:
```suggestion
import React, { Children, PropsWithChildren, type ReactElement, useEffect } from "react";
```

I'm so sorry you can ignore all of these